Source/WebCore/platform/graphics/MediaPlayer.cpp

```diff
@@ -62,6 +62,12 @@
 #endif
 #endif // USE(GSTREAMER)
 
+#if PLATFORM(WPE) && ENABLE(ENCRYPTED_MEDIA_V2)
```

Comment:
Are the headers needed, why?

Besides, a rebase is needed because the flags are different.

Also, what's the intention of flagging out with WPE? Better to flag for USE(GSTREAMER) (I don't guarantee that the flag is right but there are plenty of examples at the code.